src/audio/volume/volume_ipc4.c

```diff
@@ -147,6 +147,7 @@ int volume_init(struct processing_module *mod)
 		return -ENOMEM;
 	}
 
+	memset(cd->peak_vol, 0, vol_size);
```

Comment:
just `rzalloc()` above. BTW, is the `.peak_vol` allocation needed if `CONFIG_COMP_PEAK_VOL` is undefined?

Reply:
It's spread to nearly every volume source file, I'll add it to another commit.

Reply:
We have always for IPC4 systems CONFIG_COMP_PEAK_VOL=y, I don't think it makes sense to make a quite large patch in rush 2.12. It would make sense if we would have another volume component for ChromeOS and Linux those do not use the peak values reporting from volume.